src/distances/strings.rs

```diff
@@ -43,9 +44,40 @@ pub fn levenshtein<T: Number, U: Number>(x: &[T], y: &[T]) -> U {
     }
 }
 
+// Wrapper function for calculating distance based on NW-aligned sequences (returns edit distance only)
+pub fn needleman_wunsch<T: Number, U: Number>(x: &[T], y: &[T]) -> U {
```

Comment:
Add proper doc-strings for this function and add a link to an explanation of the distance function, e.g. an online tool or a Wikipedia page etc.